cmd/extract.go

```diff
@@ -37,9 +37,12 @@ func newExtractCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var extractCmd = &cobra.Command{
 		Use:   "extract",
-		Short: "Extract the stack and your Appsody project to a local directory",
-		Long: `This copies the full project, stack plus app, into a local directory
-in preparation to build the final container image.`,
+		Short: "Extract your Appsody project to a local directory.",
+		Long: `Extract the full application (the stack and your Appsody project), into a local directory, in preparation to build the final container image.
+		
+By default, your project gets extracted into your local '$HOME/.appsody/extract' directory, unless specified otherwise.`,
+		Example: `  appsody extract --target-dir $HOME/my-extract/directory
+  Extracts your project from the container to the local '$HOME/my-extract/directory' on your local system.`,
```

Comment:
Two local's in one sentence is confusing I'm not sure which is the most appropriate to remove but if it's a local directory it must be on your local system... so perhaps the second occurrence should be removed?

Reply:
I agree, it makes me think you can have local and remote directories on your local system.